============Tue May 30 00:59:01 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 34, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 32, in game_circulation
    level_page(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 54, in level_page
    game.bag1.event_handle(game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 159, in event_handle
    self.on_mouse_click(mouse.button_down,game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 130, in on_mouse_click
    game.next_level()
  File "D:/Python环境/Frog_Adventure/main.py", line 68, in next_level
    level_init(self)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 139, in level_init
    round_init(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 192, in round_init
    game.diceTable.round_init(game.bag1)
  File "D:\Python环境\Frog_Adventure\props\diceTable.py", line 30, in round_init
    dice = bag.take_out_dice()
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 88, in take_out_dice
    self.dice_list[index].where = "diceTable"
IndexError: list index out of range




============Tue May 30 01:00:19 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 34, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 32, in game_circulation
    level_page(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 54, in level_page
    game.bag1.event_handle(game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 159, in event_handle
    self.on_mouse_click(mouse.button_down,game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 130, in on_mouse_click
    game.next_level()
  File "D:/Python环境/Frog_Adventure/main.py", line 68, in next_level
    level_init(self)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 139, in level_init
    round_init(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 192, in round_init
    game.diceTable.round_init(game.bag1)
  File "D:\Python环境\Frog_Adventure\props\diceTable.py", line 30, in round_init
    dice = bag.take_out_dice()
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 88, in take_out_dice
    self.dice_list[index].where = "diceTable"
IndexError: list index out of range




============Tue May 30 01:08:03 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 34, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 32, in game_circulation
    level_page(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 54, in level_page
    game.bag1.event_handle(game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 159, in event_handle
    self.on_mouse_click(mouse.button_down,game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 130, in on_mouse_click
    game.next_level()
  File "D:/Python环境/Frog_Adventure/main.py", line 68, in next_level
    level_init(self)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 139, in level_init
    round_init(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 194, in round_init
    game.diceTable.round_init(game.bag1)
  File "D:\Python环境\Frog_Adventure\props\diceTable.py", line 30, in round_init
    dice = bag.take_out_dice()
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 88, in take_out_dice
    self.dice_list[index].where = "diceTable"
IndexError: list index out of range




============Tue May 30 01:11:26 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 34, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 32, in game_circulation
    level_page(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 54, in level_page
    game.bag1.event_handle(game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 159, in event_handle
    self.on_mouse_click(mouse.button_down,game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 130, in on_mouse_click
    game.next_level()
  File "D:/Python环境/Frog_Adventure/main.py", line 68, in next_level
    level_init(self)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 139, in level_init
    round_init(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 194, in round_init
    game.diceTable.round_init(game.bag1)
  File "D:\Python环境\Frog_Adventure\props\diceTable.py", line 31, in round_init
    dice = bag.take_out_dice()
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 88, in take_out_dice
    self.dice_list[index].where = "diceTable"
IndexError: list index out of range




============Tue May 30 01:14:08 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 34, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 32, in game_circulation
    level_page(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 54, in level_page
    game.bag1.event_handle(game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 162, in event_handle
    self.on_mouse_click(mouse.button_down,game)
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 133, in on_mouse_click
    game.next_level()
  File "D:/Python环境/Frog_Adventure/main.py", line 68, in next_level
    level_init(self)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 139, in level_init
    round_init(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 194, in round_init
    game.diceTable.round_init(game.bag1)
  File "D:\Python环境\Frog_Adventure\props\diceTable.py", line 30, in round_init
    dice = bag.take_out_dice()
  File "D:\Python环境\Frog_Adventure\props\bag.py", line 91, in take_out_dice
    self.dice_list[index].where = "diceTable"
IndexError: list index out of range




============Mon May 29 23:17:03 2023============

Traceback (most recent call last):
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\main.py", line 31, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameCirculation.py", line 13, in game_circulation
    game.onlineListen.join()
AttributeError: 'Game' object has no attribute 'onlineListen'




============Mon May 29 23:17:32 2023============

Traceback (most recent call last):
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\main.py", line 31, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameCirculation.py", line 13, in game_circulation
    game.onlineListen.join()
AttributeError: 'Game' object has no attribute 'onlineListen'




============Mon May 29 23:17:55 2023============

Traceback (most recent call last):
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\main.py", line 31, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameCirculation.py", line 13, in game_circulation
    game.onlineListen.join()
AttributeError: 'Game' object has no attribute 'onlineListen'




============Mon May 29 23:44:50 2023============

Traceback (most recent call last):
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\main.py", line 31, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameCirculation.py", line 13, in game_circulation
    game.onlineListen.join()
AttributeError: 'Game' object has no attribute 'onlineListen'




============Tue May 30 01:25:03 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 36, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 36, in game_circulation
    button.eventHandle()
  File "D:\Python环境\Frog_Adventure\props\button.py", line 35, in eventHandle
    self.onClick(self.game.mouse)
  File "D:\Python环境\Frog_Adventure\props\button.py", line 31, in onClick
    self.func(self.game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 110, in online
    online_init(game, "online")
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 160, in online_init
    select_Server(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 202, in select_Server
    data, address = game.listener.recvfrom(1024)
socket.timeout: timed out




============Tue May 30 01:25:16 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 36, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 36, in game_circulation
    button.eventHandle()
  File "D:\Python环境\Frog_Adventure\props\button.py", line 35, in eventHandle
    self.onClick(self.game.mouse)
  File "D:\Python环境\Frog_Adventure\props\button.py", line 31, in onClick
    self.func(self.game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 110, in online
    online_init(game, "online")
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 160, in online_init
    select_Server(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 202, in select_Server
    data, address = game.listener.recvfrom(1024)
socket.timeout: timed out




============Tue May 30 01:25:02 2023============

Traceback (most recent call last):
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\main.py", line 36, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameCirculation.py", line 36, in game_circulation
    button.eventHandle()
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\props\button.py", line 35, in eventHandle
    self.onClick(self.game.mouse)
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\props\button.py", line 31, in onClick
    self.func(self.game)
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameInit.py", line 110, in online
    online_init(game, "online")
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameInit.py", line 160, in online_init
    select_Server(game)
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameInit.py", line 200, in select_Server
    game.broadcast.sendto(json.dumps(str(game.search_win.cnt)).encode('utf-8'), (game.targetIP, 10131))
socket.gaierror: [Errno 11001] getaddrinfo failed




============Tue May 30 01:25:12 2023============

Traceback (most recent call last):
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\main.py", line 36, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameCirculation.py", line 36, in game_circulation
    button.eventHandle()
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\props\button.py", line 35, in eventHandle
    self.onClick(self.game.mouse)
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\props\button.py", line 31, in onClick
    self.func(self.game)
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameInit.py", line 110, in online
    online_init(game, "online")
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameInit.py", line 160, in online_init
    select_Server(game)
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameInit.py", line 202, in select_Server
    data, address = game.listener.recvfrom(1024)
socket.timeout: timed out




============Tue May 30 02:11:37 2023============

Traceback (most recent call last):
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\main.py", line 36, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\zhuomian\课程相关2022-2023-2\程序设计\蛙蛙勇闯地牢\buildingTool\gameCirculation.py", line 31, in game_circulation
    game.monsters[0].balls["HP"].death = True
IndexError: list index out of range




============Tue May 30 04:02:07 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 36, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 44, in game_circulation
    online_page(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 93, in online_page
    character_movement(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 181, in character_movement
    print('game.opponentAction["ATTACK"]',game.opponentAction["ATTACK"])
TypeError: string indices must be integers




============Tue May 30 04:11:53 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 36, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 44, in game_circulation
    online_page(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 93, in online_page
    character_movement(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 181, in character_movement
    print('game.opponentAction["ATTACK"]',game.opponentAction["ATTACK"])
TypeError: string indices must be integers




============Tue May 30 04:32:52 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 36, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 42, in game_circulation
    button.eventHandle()
  File "D:\Python环境\Frog_Adventure\props\button.py", line 35, in eventHandle
    self.onClick(self.game.mouse)
  File "D:\Python环境\Frog_Adventure\props\button.py", line 31, in onClick
    self.func(self.game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 109, in online
    online_init(game, "online")
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 159, in online_init
    select_Server(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 206, in select_Server
    game.broadcast.sendto(json.dumps(str(game.search_win.cnt)).encode('utf-8'), (game.targetIP, 10131))
socket.gaierror: [Errno 11001] getaddrinfo failed




============Tue May 30 04:34:03 2023============

Traceback (most recent call last):
  File "D:/Python环境/Frog_Adventure/main.py", line 36, in __init__
    game_circulation(self)  # 游戏内部循环中的事件处理
  File "D:\Python环境\Frog_Adventure\buildingTool\gameCirculation.py", line 42, in game_circulation
    button.eventHandle()
  File "D:\Python环境\Frog_Adventure\props\button.py", line 35, in eventHandle
    self.onClick(self.game.mouse)
  File "D:\Python环境\Frog_Adventure\props\button.py", line 31, in onClick
    self.func(self.game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 109, in online
    online_init(game, "online")
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 159, in online_init
    select_Server(game)
  File "D:\Python环境\Frog_Adventure\buildingTool\gameInit.py", line 206, in select_Server
    game.broadcast.sendto(json.dumps(str(game.search_win.cnt)).encode('utf-8'), (game.targetIP, 10131))
socket.gaierror: [Errno 11001] getaddrinfo failed




